App/UI/GreenCertificate/ModalDetail/GreenCertificateVaccineDetail/GreenCertificateVaccineDetailView.swift

```diff
@@ -17,755 +17,797 @@ import Models
-    case BharatBiotech = "Bharat-Biotech"
-    
-    case EU_1_20_1528 = "EU/1/20/1528"
-    case EU_1_20_1507 = "EU/1/20/1507"
-    case EU_1_21_1529 = "EU/1/21/1529"
-    case EU_1_20_1525 = "EU/1/20/1525"
-    case CVnCoV = "CVnCoV"
-    case Sputnik_V = "Sputnik-V"
-    case Convidecia = "Convidecia"
-    case EpiVacCorona = "EpiVacCorona"
-    case BBIBP_CorV = "BBIBP-CorV"
-    case Inactivated_SARS_CoV_2_Vero_Cell = "Inactivated-SARS-CoV-2-Vero-Cell"
-    case CoronaVac = "CoronaVac"
-    case Covaxin = "Covaxin"
-    
-    func getDescription() -> String{
-        switch self {
-        case .ORG_100001699:
-            return "AstraZeneca AB"
-        case .ORG_100030215:
-            return "Biontech Manufacturing GmbH"
-        case .ORG_100001417:
-            return "Janssen-Cilag International"
-        case .ORG_100031184:
-            return "Moderna Biotech Spain S.L."
-        case .ORG_100006270:
-            return "Curevac AG"
-        case .ORG_100013793:
-            return "CanSino Biologics"
-        case .ORG_100020693:
-            return "China Sinopharm International Corp. - Beijing location"
-        case .ORG_100010771:
-            return "Sinopharm Weiqida Europe Pharmaceutical s.r.o. - Prague location"
-        case .ORG_100024420:
-            return "Sinopharm Zhijun (Shenzhen) Pharmaceutical Co. Ltd. - Shenzhen location"
-        case .ORG_100032020:
-            return "Novavax CZ AS"
-        case .GamaleyaResearchInstitute:
-            return "Gamaleya Research Institute"
-        case .VectorInstitute:
-            return "Vector Institute"
-        case .SinovacBiotech:
-            return "Sinovac Biotech"
-        case .BharatBiotech:
-            return "Bharat Biotech"
-        case .EU_1_20_1528:
-            return "Comirnaty"
-        case .EU_1_20_1507:
-            return "COVID-19 Vaccine Moderna"
-        case .EU_1_21_1529:
-            return "Vaxzevria"
-        case .EU_1_20_1525:
-            return "COVID-19 Vaccine Janssen"
-        case .CVnCoV:
-            return "CVnCoV"
-        case .Sputnik_V:
-            return "Sputnik-V"
-        case .Convidecia:
-            return "Convidecia"
-        case .EpiVacCorona:
-            return "EpiVacCorona"
-        case .BBIBP_CorV:
-            return "BBIBP-CorV"
-        case .Inactivated_SARS_CoV_2_Vero_Cell:
-            return "Inactivated SARS-CoV-2 (Vero Cell)"
-        case .CoronaVac:
-            return "CoronaVac"
-        case .Covaxin:
-            return "Covaxin (also known as BBV152 A, B, C)"
-        }
+public enum VaccineName: String {
+  case ORG_100001699 = "ORG-100001699"
+  case ORG_100030215 = "ORG-100030215"
+  case ORG_100001417 = "ORG-100001417"
+  case ORG_100031184 = "ORG-100031184"
+  case ORG_100006270 = "ORG-100006270"
+  case ORG_100013793 = "ORG-100013793"
+  case ORG_100020693 = "ORG-100020693"
+  case ORG_100010771 = "ORG-100010771"
+  case ORG_100024420 = "ORG-100024420"
```

Comment:
<!--
  1 failure:  swiftlint (0.39.2...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: App/UI/GreenCertificate/ModalDetail/GreenCertificateVaccineDetail/GreenCertificateVaccineDetailView.swift;
  Line: 737;
-->
- :no_entry_sign: swiftlint (0.39.2): Enum element name should only contain alphanumeric characters: 'ORG_100024420' (identifier_name)